packages/mdc-textfield/README.md

```diff
@@ -285,13 +285,13 @@ can be controlled by passing a ripple factory argument to the constructor.
 
 ```js
 const textFieldBoxEl = document.querySelector('.mdc-text-field--box');
-const textField = new MDCTextField(textFieldBoxEl, /* foundation */ undefined, (el) => {
-  // do something with el...
-  return new MDCRipple(el);
+const textField = new MDCTextField(textFieldBoxEl, /* foundation */ undefined, (el, foundation) => {
```

Comment:
My only request is that we make it more clear that we mean the ripple foundation and not the component foundation. Otherwise LGTM

Reply:
Done.